db/seeds.rb

```diff
@@ -5,3 +5,10 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'nokogiri'
+require 'open-uri'
+doc = Nokogiri::HTML(open("http://blogs.transparent.com/danish/2013/10/31/100-most-common-written-words-in-danish/"))
+
+doc.css('div > p + table tr')[1..100].each do |x|
+  Card.create(original: x.css('td')[1].text.strip, translated: x.css('td')[2].text.strip)
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.